src/Sylius/Bundle/ThemeBundle/Command/AssetsInstallCommand.php

```diff
@@ -65,7 +65,13 @@ protected function execute(InputInterface $input, OutputInterface $output): void
             $symlinkMask = max($symlinkMask, AssetsInstallerInterface::RELATIVE_SYMLINK);
         }
 
-        $assetsInstaller->installAssets($input->getArgument('target'), $symlinkMask);
+        $targetDir = $this->getContainer()->getParameter('sylius_core.public_dir');
+
+        if ($input->getArgument('target')) {
+            $targetDir = $input->getArgument('target');
+        }
```

Comment:
We could extract this to a separate private function to make it even more clean & clear, wdyt?

```php
$assetsInstaller->installAssets($this->getTargetDir($input), $symlinkMask);

// ....


private function getTargetDir(InputInterface $input): string { ... }
```